notebooks/228-clip-zero-shot-image-classification/228-clip-zero-shot-image-classification.ipynb

```diff
@@ -239,56 +259,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import ipywidgets as widgets\n",
-    "style = {'description_width': 'initial'}\n",
-    "\n",
-    "image_widget = widgets.FileUpload(\n",
-    "    accept='',\n",
-    "    multiple=False,\n",
-    "    description='Upload image',\n",
-    "    style=style\n",
-    ")\n",
+    "import gradio as gr\n",
+    "\n",
     "\n",
-    "labels_widget = widgets.Textarea(\n",
-    "    value='cat,dog,bird',\n",
-    "    placeholder='Type something',\n",
-    "    description='Enter your classes separated by ,:',\n",
-    "    disabled=False,\n",
-    "    style=style\n",
+    "def classify(image, text):\n",
```

Comment:
Do we need to add a doc string for `classify` utility function?

Reply:
Makes sense. Added